lib/pltr/v2/reducers/tours.js

```diff
@@ -8,56 +9,64 @@ export default function tours(state = INITIAL_STATE, action) {
     case SET_TOUR_FEATURE:
       const newState = Object.assign({}, INITIAL_STATE)
       newState.feature = action.payload //an object with feature: 'string', featureId: int
+      newState.steps = ACTS_TOUR_STEPS
```

Comment:
If the idea is to pass the tour steps into redux when we initialise the tour, then we need to instead have these steps be part of the action.

Reply:
It's actually more efficient to keep the steps out of redux, it's only important to store the {feature} there, and I've moved the steps back into the Tour component to avoid circular dependency and storing the entire steps array in redux. Thanks for pointing this out!